pom.xml

```diff
@@ -270,6 +270,7 @@
                     <artifactId>maven-javadoc-plugin</artifactId>
                     <version>${version.plugin.javadoc}</version>
                     <configuration>
+                        <source>8</source>
```

Comment:
This gives me pause.  How come?

Reply:
Without it you get module errors from javadoc about stuff referencing stuff in the automatic module.